.github/docker-compose/docker-compose.prod.yml

```diff
@@ -1,12 +1,28 @@
 version: "3"
 services:
-    backend:
+    backendAction:
         build: ../..
         image: openslides-backend
         ports:
             - "9002:9002"
+        environment:
+            - OPENSLIDES_BACKEND_COMPONENT=action
+            - DATASTORE_READER_HOST=reader
+            - DATASTORE_READER_PORT=9010
+            - DATASTORE_WRITER_HOST=writer
+            - DATASTORE_WRITER_PORT=9011
+        depends_on:
+            - writer
+            - reader
+        secrets:
+            - postgres_password
+    backendPresenter:
+        build: ../..
+        image: openslides-backend
+        ports:
             - "9003:9003"
         environment:
+            - OPENSLIDES_BACKEND_COMPONENT=presenter
```

Comment:
No, this was probably missed in https://github.com/OpenSlides/openslides-backend/pull/1254. I will remove it

Reply:
It is not as easy to remove it as I thought, since it is tethered to the entrypoint which is always executed in the backend. So I would propose to leave it as-is.